src/api/search-service-api/concepts.ts

```diff
@@ -0,0 +1,74 @@
+import { searchServiceApiGet } from './host';
+import { normalizeAggregations } from '../../lib/normalizeAggregations';
+import { Concept } from '../../types';
+
+const mapSorting = ({ sortfield }: any) =>
+  sortfield === 'harvest.firstHarvested'
+    ? { field: 'harvest.firstHarvested', direction: 'desc' }
+    : undefined;
+
+const mapFilters = ({
+  id,
+  identifiers,
+  uri,
+  orgPath,
+  last_x_days,
+  seeAlso
+}: any) => {
+  const filters = [];
+  if (id) {
+    filters.push({ _id: id });
+  }
+  if (identifiers) {
+    filters.push({
+      collection: {
+        field: 'identifier.keyword',
+        values: identifiers
+      }
+    });
+  }
+  if (uri) {
+    filters.push({
+      collection: {
+        field: 'uri',
+        values: uri
+      }
+    });
+  }
```

Comment:
Denne finnes, men har nå en helt annen struktur
```suggestion
```

Reply:
Jeg har merget min PR der alt dette er slettet (refakturert), blir forhåpentligvis ikke for mye konflikter 🤞